### PR TITLE
Bugfix/default transcripts

### PR DESCRIPTION
**Pull request by OPersian**
* Fix default transcripts upload (when no standard transcript is uploaded, enabled transcripts are now updated on available transcript upload).
* Remove all enabled transcripts on standard transcripts reset.
* Display default subs of distinct languages only.
* Hidden enabled default subs are not duplicated on subs manual upload.

### CODE REVIEW

**Comment by z4y4ts** [comment]
Please write better description for this PR. What does it fix exactly?

File: video_xblock/backends/base.py [hunk 1]
```diff
@@ -297,9 +300,29 @@ def get_transcript_language_parameters(lang_code):
...
+
+        Arguments:
+            default_transcripts (list): Nested list of dictionaries with data on default transcripts.
+        Returns:
+            distinct_transcripts (list): Distinct default transcripts to be shown in studio editor.
+        """
+        get_values = operator.itemgetter('lang')
+        default_transcripts.sort(key=get_values)
+        distinct_transcripts = []
+        for key, group in itertools.groupby(default_transcripts, get_values):  # pylint: disable=unused-variable
```
Comment (z4y4ts) [inline video_xblock/backends/base.py]: `key` -> `_key` or `_`. So you don't have to disable pylint

File: video_xblock/static/html/studio_edit.html [hunk 2]
```diff
@@ -141,14 +141,14 @@
               <div class="custom-field-list">
 
                 <div class="custom-field-section-label main-section-label">{% trans "Show default transcripts" %}</div>
-                   <input type="checkbox" class="default-transcripts-switch-input" id="default-transcript-onoff-switch">
+                   <input type="checkbox" class="default-transcripts-switch-input" id="default-transcript-onoff-switch" checked>
```
Comment (xahgmah) [inline video_xblock/static/html/studio_edit.html]: Why does `checked` parameter hardcoded? I think it should consider some variable(s)

File: video_xblock/static/js/studio-edit.js [hunk 3]
```diff
@@ -12,6 +12,11 @@ function StudioEditableXBlock(runtime, element) {
     // Studio includes a copy of tinyMCE and its jQuery plugin
     var tinyMceAvailable = (typeof $.fn.tinymce !== 'undefined');  // TODO: Remove TinyMCE
     var datepickerAvailable = (typeof $.fn.datepicker !== 'undefined'); // Studio includes datepicker jQuery plugin
+    var $enabledLabel = $('div.custom-field-section-label:contains("Enabled transcripts")');
```
Comment (xahgmah) [inline video_xblock/static/js/studio-edit.js]: Very bad selector. Titles will support translations in a feature. Please use some className for the selector.

File: video_xblock/static/html/studio_edit.html (hunk 2 above)
Comment (OPersian) [inline video_xblock/static/html/studio_edit.html]: Changed. Though a user should define whether to show default transcripts or not (will be considered later on), subs are now hidden by default.

File: video_xblock/static/js/studio-edit.js (hunk 3 above)
Comment (OPersian) [inline video_xblock/static/js/studio-edit.js]: Got rid of such selectors. Thanks!

File: video_xblock/static/css/studio-edit.css [hunk 4]
```diff
@@ -167,12 +167,12 @@
...
 }
 
-.default-transcripts-wrapper {
-    display: none;
-}
-.default-transcripts-switch-input:checked ~ .default-transcripts-wrapper {
+.default-transcripts-wrapper:checked {
     display: block;
 }
+.default-transcripts-switch-input:not(:checked) ~ .default-transcripts-wrapper {
```
Comment (OPersian) [inline video_xblock/static/css/studio-edit.css]: It serves to affect default subs display (show/hide `.default-transcripts-wrapper`) on toggle-switch click (`.default-transcripts-switch-input`).

**Pull request merged**